README.md

```diff
@@ -0,0 +1,28 @@
+# BaNaNaS CLI frontend
+
+[![GitHub License](https://img.shields.io/github/license/OpenTTD/bananas-frontend-cli)](https://github.com/OpenTTD/bananas-frontend-cli/blob/master/LICENSE)
+[![GitHub Tag](https://img.shields.io/github/v/tag/OpenTTD/bananas-frontend-cli?include_prereleases&label=stable)](https://github.com/OpenTTD/bananas-frontend-cli/releases)
+[![GitHub commits since latest release](https://img.shields.io/github/commits-since/OpenTTD/bananas-frontend-cli/latest/master)](https://github.com/OpenTTD/bananas-frontend-cli/commits/master)
+
+[![GitHub Workflow Status (Testing)](https://img.shields.io/github/workflow/status/OpenTTD/bananas-frontend-cli/Testing/master?label=master)](https://github.com/OpenTTD/bananas-frontend-cli/actions?query=workflow%3ATesting)
+
+This is a CLI frontend for the OpenTTD's content service, called BaNaNaS.
+
+## Development
+
+This API is written in Python 3.8 with aiohttp, and makes strong use of asyncio.
+
+## Usage
+
+To start it, you are advised to first create a virtualenv:
+
+```bash
+python3 -m venv .env
+.env/bin/pip install -r requirements.txt
+```
+
+Next, check out the help.
+
+```bash
+.env/bin/python -m bananas_cli --help
+```
```

Comment:
Is click supposed to show description of the command args somewhere?
It does not for me.

Reply:
Ah, it does with "command -h". But for some reason it wants authentication / server connection just to print the help message.